src/modules/auth/entity.rs

```diff
@@ -2,6 +2,6 @@ use async_graphql::SimpleObject;
 use serde::{Deserialize, Serialize};
 
 #[derive(Debug, Deserialize, Serialize, SimpleObject)]
-pub struct Token {
-    pub token: String,
+pub struct Tokens {
+    pub access_token: String,
```

Comment:
As we progress on adding more API capabilities, `Tokens` will hold `refresh_token` along with `access_token`.